users/views.py

```diff
@@ -248,3 +247,56 @@ def register_confirm(request, activation_key):
         'users/confirm.html',
         {'username':user.username},
         context_instance=RequestContext(request, processors=[custom_proc]))
+
+def notification(request):
+    try:
+        ur_notifications = Notification.objects.filter \
+        (reciver=request.user, read=False).order_by('-id')
```

Comment:
When the filter find nothing, it will return an empty list.
I don't it will cause exception.